server/src/main/java/org/kaazing/gateway/server/config/parse/GatewayConfigNamespace.java

```diff
@@ -111,12 +105,8 @@ public String toURI() {
                 uri = NS_SEPTEMBER_2014_URI;
                 break;
 
-            case NOVEMBER_2015:
-                uri = NS_NOVEMBER_2015_URI;
-                break;
-                
             case CURRENT_NS:
-                uri = NS_JUNE_2016_URI;
+                uri = NS_NOVEMBER_2015_URI;
```

Comment:
Hi,

This was deleted by @dpwspoon in this commit with the comment: 
Revert "Login Module fix". I have simply included his change. I assume that he reverted all the changes done by that module. 

Regards,
Doru